conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -18,15 +24,28 @@ class GramineEnclaveHandle(
     override val enclaveClassName: String,
     private val zipFileUrl: URL
 ) : EnclaveHandle {
-    private lateinit var processGramineDirect: Process
+
+    companion object {
+        private val ENCLAVE_MODE_EXECUTABLES = mapOf(
```

Comment:
This is better served as a function with a `when` statement. Saves on keeping the Map around.